### PR TITLE
[plugin.video.newyankeeworkshop] 0.0.2

### PR DESCRIPTION
**Pull request by th0ma5w**
### Description
Corrected video extraction code for source website. Formerly the site made an MP4 video available, but now it provides an .m3u8 playlist. Additionally, a requests library session is now used for maintaining cookies during interaction with the source site. Thank you all for you continued hard work on Kodi!

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.newyankeeworkshop/addon.xml [hunk 1]
```diff
@@ -12,13 +12,17 @@
...
-	    - Initial Release</news>
+<news>
+v0.0.2 (05.17.2020)
+  - Updates to video extraction code
+v0.0.1 (09.09.2018)
+  - Initial Release
+</news>
     <language>en</language>
     <platform>all</platform>
     <license>GNU GENERAL PUBLIC LICENSE. Version 3, June 2007</license>
```
Comment (enen92) [inline plugin.video.newyankeeworkshop/addon.xml:23]: Can you use a spdx identifier for the license? Should be GPL-3.0-only or GPL-3.0-or-later depending on your license

**Review by enen92 — APPROVED** [review]
Just the license thing (which is a minor) and is good to go

File: plugin.video.newyankeeworkshop/addon.xml (hunk 1 above)
Comment (th0ma5w) [reply plugin.video.newyankeeworkshop/addon.xml]: Updated license to GPL-3.0-or-later ... thank you!!

**Comment by enen92** [comment]
Sorry for the delay merging it, and thanks for he ping in irc

**Comment by th0ma5w** [comment]
Thank you for all your time with this!!

**Comment by th0ma5w** [comment]
Just a heads up, the underlying website that drove this has disappeared, there is talk on other sites about general licensing changes with this content and the various groups related to it, and their respective websites. I will monitor this for a couple weeks, but it may just be gone for now.

**Pull request merged**